src/Cli/dotnet/Program.cs

```diff
@@ -29,55 +30,91 @@ public static int Main(string[] args)
         {
             DebugHelper.HandleDebugSwitch(ref args);
 
-            new MulticoreJitActivator().TryActivateMulticoreJit();
+            // Capture the current timestamp to calculate the host overhead.
+            DateTime mainTimeStamp = DateTime.Now;
 
-            if (Env.GetEnvironmentVariableAsBool("DOTNET_CLI_CAPTURE_TIMING", false))
+            // Determine if the perf log is enabled.
+            bool perfLogEnabled = Env.GetEnvironmentVariableAsBool("DOTNET_CLI_PERF_LOG", false);
+            PerformanceLogStartupInformation startupInfo = null;
+            if (perfLogEnabled)
             {
-                PerfTrace.Enabled = true;
-            }
+                // Initialize the performance log start-up information.
```

Comment:
nit: remove unnecessary comment. 
"Initialize the performance log start-up information.",

Reply:
Removed